src/include/network/lan/address.rb

```diff
@@ -1631,5 +1611,36 @@ def repeated_physical_port_ids(slaves)
 
       physical_port_ids
     end
+
+    # Performs hostname update
```

Comment:
this definitively needs more elaboration what it do as I see many side effect like that 

1. depends on current LangItems.ipaddr
2. takes original hostname for currnet ipaddr
3. it removes any hostnames for current ipaddr
4. what happens with original hostname for original ipaddr?
5. it adds new hostname to new upaddr, but only if hostname is not empty
6. behavior when hostname is empty is not documented

Reply:
documentation improved